README.md

```diff
@@ -15,7 +15,7 @@ The original FFXIV launcher is slow, tedious, kinda ugly and cannot save your pa
 * Integrity check for game files
 * Chat filtering
 * Chat bridge to Discord
-* Discord notifications for Duty Finder, fates, retainer sales, etc.
+* Discord notifications for fates, retainer sales, etc.

```

Comment:
Actually it's Duty Finder, retainer sales, etc., but I'll fix it myself later, thanks 👍